sql/dql/basic-queries/select.md

```diff
@@ -40,14 +51,25 @@ FROM pokemon;
 
 The above command gets all of the *columns* and all of the *rows* from the `pokemon` table.
 
-We can also use the `SELECT` statement to specify the exact *columns* we want to keep from the table, and to discard the others.
+We can also use the `SELECT` statement to specify the exact *columns* we want to keep from the table, and to discard the others. Depending on how the database was created, you might have to deal with complicated column names. In order to increase readability, the columns can be renamed using *aliases* represented by the `AS` keyword.
```

Comment:
Modified line 54 to say:
>Depending on how the database was created, you might have to deal with complicated column names. In order to increase readability, or to avoid repeatedly writing lengthy names, the columns can be renamed using *aliases* represented by the `AS` keyword.